projects/plugins/jetpack/extensions/blocks/ai-assistant/extensions/jetpack-contact-form/components/ai-assistant-popover/index.tsx

```diff
@@ -60,44 +59,52 @@ function getSerializedContentFromBlock( clientId: string ): string {
 export const AiAssistantPopover = ( {
 	clientId = '',
 }: AiAssistantPopoverProps ): React.ReactNode => {
-	const { isVisible, hide, toggle, popoverProps, inputValue, setInputValue } =
+	const { isVisible, hide, toggle, popoverProps, inputValue, setInputValue, width } =
 		useContext( AiAssistantUiContext );
 
 	const { requestSuggestion, requestingState } = useAiContext();
 
-	const isDisabled = requestingState === 'requesting' || requestingState === 'suggesting';
+	const isLoading = requestingState === 'requesting' || requestingState === 'suggesting';
+
+	const placeholder = __( 'Which form do you need?', 'jetpack' );
+
+	const loadingPlaceholder = __( 'Creating your form. Please wait a few moments.', 'jetpack' );
```

Comment:
Is it part of the design? What happens when the user edits the form?

Reply:
For a more generic placeholder, we could use "Processing, please wait..."

Reply:
When the user asks for "translate to Spanish" definitively we are not creating the form, from the PoV of the user, but editing it, IMO.